Adventure/Adventure.sln

```diff
@@ -13,35 +13,28 @@ Global
 		Release|iPhoneSimulator = Release|iPhoneSimulator
 		Debug|iPhone = Debug|iPhone
 		Release|iPhone = Release|iPhone
-		Ad-Hoc|iPhone = Ad-Hoc|iPhone

```

Comment:
The below changes are resulting in some discrepancies in the build configurations when the solution is opened from VSFMac. 
https://screencast.com/t/ByUKxPHpi . 
1. When Mac sample is set to start up project ,  user must able to set to Debug|MyMac and Release|MyMac
2. When iOS sample is set to start up project,  user must able to set to   Debug|iPhone , Release|iPhone, Debug|iPhoneSimulator, Release|iPhoneSimulator
 1 and 2 seems to be interchanged for mac and ios samples and ios samples cannot be deployed to simulator. 
Please take a look at the screencast.  There are similar issues in other samples.